20110126
========

- watch_tim.
  When :mod:`lino.utils.daemoncommand` has been told to 
  create a pidfile, and when that pidfile already exists
  (e.g. after a server crash, or when the previous watch_tim 
  process has been terminated using `kill -9`), 
  the the new daemon waited 
  dann wartete der Prozess bisher geduldig *ad eternam*, 
  dass die Datei verschwinden möge.
  Was sie natürlich nur tat, wenn der Systemverwalter 
  sie manuell löschte.
  Damit letzter das Problem mitgeteilt bekommt, 
  setzen wir einen 
  `Timeout 
  <http://packages.python.org/lockfile/lockfile.html#lockfile.FileLock.acquire>`_
  von 1 Sekunde.
  
  We are one step further
  One problem remains.
  
  


